### PR TITLE
improving sender chat docstring

### PR DESCRIPTION
**Pull request by Poolitzer**
also adding a note to channel edge case

### CODE REVIEW

File: telegram/ext/filters.py [hunk 1]
```diff
@@ -1681,16 +1681,22 @@ class sender_chat(_ChatUserBaseFilter):
...
               ``@anonymous``, use
               ``MessageHandler(Filters.sender_chat(username='anonymous'), callback_method)``.
-            * To filter for messages forwarded from *any* channel, use
+            * To filter for messages forwarded to a discussion group from *any* channel, use
               ``MessageHandler(Filters.sender_chat.channel, callback_method)``.
             * To filter for messages of anonymous admins in *any* super group, use
               ``MessageHandler(Filters.sender_chat.super_group, callback_method)``.
 
+        Note:
+            Remember, sender_chat is also set for messages in a channel as the channel itself,
```
Comment (Bibo-Joshi) [inline telegram/ext/filters.py:1695]: Please render sender_chat as monospace

File: telegram/ext/filters.py [hunk 2]
```diff
@@ -1681,16 +1681,22 @@ class sender_chat(_ChatUserBaseFilter):
...
+            so when your bot is an admin in a channel and the linked discussion group and you
+            filter messages from this channel, you would receive the message twice (once from
```
Comment (Bibo-Joshi) [inline telegram/ext/filters.py:1697]: The "and you filter messages from this channel" seems a bit superfluous to me

Comment (Poolitzer) [reply telegram/ext/filters.py:1697]: agreed

File: telegram/ext/filters.py (hunk 1 above)
Comment (Poolitzer) [reply telegram/ext/filters.py:1695]: okay chef

**Pull request merged**